example/android/build.gradle

```diff
@@ -1,11 +1,11 @@
 buildscript {
     ext {
         buildToolsVersion = "34.0.0"
-        minSdkVersion = 21
+        minSdkVersion = 23
```

Comment:
is it because of the latest RN version?

Reply:
Yes `com.facebook.react:react-android:0.74.3` requires a minimum `minSdkVersion` of 23.